bpf/Makefile

```diff
@@ -78,9 +78,16 @@ MAX_BASE_OPTIONS = -DSKIP_DEBUG=1 -DENABLE_IPV4=1 -DENABLE_IPV6=1 \
 	-DMONITOR_AGGREGATION=3 -DCT_REPORT_FLAGS=0x0002
 ifeq ("$(KERNEL)","49")
 # IPSec is incompatible with BPF NodePort so we only enable on 4.9.
-MAX_BASE_OPTIONS += -DENABLE_IPSEC=1
+MAX_BASE_OPTIONS += -DENABLE_IPSEC=1 -DNEEDS_RELAX_VERIFIER=1
 else
-MAX_BASE_OPTIONS += -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1
+MAX_BASE_OPTIONS += -DHAVE_LPM_TRIE_MAP_TYPE=1 -DHAVE_LRU_HASH_MAP_TYPE=1 \
+	-DENABLE_MASQUERADE=1
+ifeq ("$(KERNEL)","netnext")
+MAX_BASE_OPTIONS += -DENABLE_TPROXY=1 -DENABLE_REDIRECT_FAST=1 -DENABLE_BANDWIDTH_MANAGER=1 \
```

Comment:
I don't think this is the correct conflict resolution as these three datapath options don't exist in v1.8.